frontend/src/components/FileUpload.js

```diff
@@ -0,0 +1,26 @@
+import {fileUpload} from "../services/fileUploadService";
+import {useState} from "react";
+
+export function FileUpload() {
```

Comment:
Maybe make the component the default export, you probably don't have other exports from this file.